File: past201912_a/Kotlin/main.kt
fun main(args: Array<String>){
    val s = readLine()
    
    for( b in s.toString().toByteArray(Charsets.UTF_8) ){
        if(!(b>=0x30&&b<=0x39)){
            println("error")
            return
        }
    }
    println(s.toString().toInt()*2)
}

File: past201912_a/Rust/main.rs
use std::io;

fn main() {
    let mut s = String::new();

    io::stdin().read_line(&mut s).ok();

    for b in s.as_bytes() {
        if !( b >=&mut 0x30 && b <=&mut 0x39 ) {
            println!("error\n");
            return;
        }
    }

	println!("{}", s.parse::<u32>()*2);
    return;
}